noether_conversions/CMakeLists.txt

```diff
@@ -15,29 +15,43 @@ include(${VTK_USE_FILE})
 find_package(Eigen3 REQUIRED)
 
 catkin_package(
-  INCLUDE_DIRS include
+  INCLUDE_DIRS
+    include
   LIBRARIES
-    noether_conversions
+    ${PROJECT_NAME}
   CATKIN_DEPENDS
     eigen_conversions
     geometry_msgs
     tf
     tool_path_planner
   DEPENDS
-    Eigen
+    Eigen3
```

Comment:
Is this a problem? It looks like cmake_modules is pulling in Eigen? I'm not really sure why we have it. Should we just be including Eigen like normal or is there a reason to use cmake_modules @jrgnicho ? It looks like the project is a bit stale

Reply:
Really do not know what to do here.  Changed to EIGEN3 here (and a few other places) so that it finds files properly.  It is unclear what function cmake_modules is performing.  I'll try building without it.

Reply:
This should be `EIGEN3`. Not `Eigen` or `Eigen3`. Case is important.

Reply:
It has been changed to EIGEN3 in the most recent version of the diff.